src/test/functional/steps/court-addresses.ts

```diff
@@ -0,0 +1,156 @@
+import {Then, Given, When} from 'cucumber';
+import {expect} from 'chai';
+
+import * as I from '../utlis/puppeteer.util';
+
+Then('I click the Addresses tab', async () => {
+  const selector = '#tab_addresses';
+  const elementExist = await I.checkElement(selector);
+  expect(elementExist).equal(true);
```

Comment:
Can do it in one line if you want (up to you): 

`expect(await I.checkElement(selector)).equal(true);`